app/stv/stv.js

```diff
@@ -0,0 +1,370 @@
+        if (fraction === 0 || vote.priorities.length === 1) continue;
+
+        // Fractional transfer. We mutate the weight for these votes by a fraction
+        vote['weight'] = vote.weight * fraction;
+        // Push the mutated votes to the list of votes to be processed in the next iteration
+        nextRoundVotes.push(vote);
+      }
+    } else {
+      // Find the lowest score
+      const minScore = Math.min(
+        ...alternatives.map(
+          (alternative) => counts[alternative.description] || 0
+        )
+      );
+
+      // Find the candidates with the lowest score
+      const minAlternatives = alternatives.filter(
+        (alternative) =>
+          (counts[alternative.description] || 0) <= minScore + EPSILON
+      );
+
+      // There is a tie for eliminating candidates. Per Scottish STV we must look at the previous rounds
+      if (minAlternatives.length > 1) {
+        let reverseIteration = iteration;
+
+        // Log the Tie
+        log.push({
+          action: 'TIE',
+          description: `There are ${
+            minAlternatives.length
+          } candidates with a score of ${Number(
+            minScore.toFixed(4)
+          )} at iteration ${reverseIteration}`,
+        });
+
+        // As long as the reveseindex is still larger then 1 we can look further back
+        while (reverseIteration >= 1) {
+          // Find the log object for the last iteration
+          const logObject = log.find(
+            (entry) => entry.iteration === reverseIteration
+          );
+
+          // Find the lowest score (with regard to the alternatives in the actual iteration)
+          const iterationMinScore = Math.min(
+            ...minAlternatives.map((a) => logObject.counts[a.description] || 0)
+          );
+
+          // Find the candidates (in regard to the actual iteration) that has the lowest score
+          const iterationMinAlternatives = minAlternatives.filter(
+            (alternative) =>
+              (logObject.counts[alternative.description] || 0) <=
+              iterationMinScore + EPSILON
+          );
+
+          // If we are at iteration lvl 1 and there is still a tie we cannot do anything
+          if (reverseIteration === 1 && iterationMinAlternatives.length > 1) {
+            log.push({
+              action: 'TIE',
+              description:
+                'The backward checking went to iteration 1 without breaking the tie',
+            });
+            // Eliminate all candidates that are in the last iterationMinAlternatives
+            log.push({
+              action: 'MULTI_TIE_ELIMINATIONS',
+              alternatives: iterationMinAlternatives,
+              minScore: Number(minScore.toFixed(4)),
+            });
+            iterationMinAlternatives.forEach(
+              (alternative) => (doneAlternatives[alternative._id] = {})
+            );
+            break;
+          }
```

Comment:
Do we here see a tie and then eliminate all the candidates in the tie? I'm confused 😵

Reply:
Here we are at iteration 1, meaning we cannot go any further back to check for differences in score. 
We still have two users in with the `iterativeMinScore` and both of these must then be eliminated. 

This was the discussion below. In conclusion, we either just declare the elction UNRESOVLED right away, or just eliminate both. If it later shows that eliminating both of these candidates made the election UNRESOLVED then be it, we could not have done anything else.
![image](https://user-images.githubusercontent.com/23152018/102518353-77d7d080-4091-11eb-8c1c-e67f195e6ba1.png)

Reply:
Aah, I see. But this won't always be just two candidates though right? And I think a comment would be nice. It's a bit confusing that there is a comment saying `we cannot do anything` and then we eliminate all the minAlternatives 🙃

Reply:
Yeah, the comment is a little confusing I agree. I think i wrote if when we still did the `return UNRESOLVED` logic, but then we chagned it